Brokerages/Zerodha/ZerodhaWebSocketClientWrapper.cs

```diff
@@ -0,0 +1,257 @@
+﻿using System;
+using System.IO;
+using System.Net.WebSockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using QuantConnect.Logging;
+using QuantConnect.Util;
+
+namespace QuantConnect.Brokerages.Zerodha
+{
+    //<summary>
+    // ZerodhaWebSocketClientWrapper ..
+    //</summary>
+    public class ZerodhaWebSocketClientWrapper
```

Comment:
Wondering if we could extend the existing `WebSocketClientWrapper` class as `ZerodhaWebSocketClientWrapper` is very similar?

Reply:
Zerodha implements a custom binary messaging format whereas `WebSocketClientWrapper` only supports reading messages in string format. May be we can extend `WebSocketClientWrapper` later to support both the scenario's.